test/functional/multiwallet.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the wallet."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+class MultiWalletTest(BitcoinTestFramework):
+
+    def __init__(self):
+        super().__init__()
+        self.setup_clean_chain = True
+        self.num_nodes = 1
+        self.extra_args = [['-wallet=w1', '-wallet=w2','-wallet=w3']]
+
+    def setup_network(self):
+        self.nodes = self.start_nodes(1, self.options.tmpdir, self.extra_args[:1])
+
+    def run_test(self):
+        w1 = self.nodes[0] / "v1/wallet/w1"
+        w1.generate(1)
+
+        #accessing default wallet must be impossible
+        assert_raises(JSONRPCException, self.nodes[0].getwalletinfo)
```

Comment:
please use the `assert_raises_jsonrpc()` function to check the error code and message